packages/sdk-react/README.md

```diff
@@ -120,6 +120,66 @@ return (
 );
 ```
 
+### Building a Dashboard Tab
+
+#### 1. Create a new widget
+
+Here is an example of creating a new widget:
+
+```tsx
+import { Button, Text } from "@fluentui/react-components";
+import { BaseWidget } from "@microsoft/teamsfx-react";
+import { SampleModel } from "../models/sampleModel";
+import { getSampleData } from "../services/sampleService";
+
+interface SampleWidgetState {
+  data?: SampleModel;
+}
+
+export class SampleWidget extends BaseWidget<any, SampleWidgetState> {
+  async getData(): Promise<SampleWidgetState> {
+    return { data: getSampleData() };
+  }
+
+  header(): JSX.Element | undefined {
```

Comment:
It might be better to add `override` keyword explicitly to indicate this is an override method.

Reply:
OK, added.